.github/workflows/update-versions.yml

```diff
@@ -0,0 +1,37 @@
+
+name: update-version
+on:
+  pull_request:
+          types: [opened]
+          branches: [master]
+
+jobs:
+  update-version-in-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+        with: 
+          ref: ${{ github.head_ref }}
+
+      - if: github.actor == "release-please[bot]"
+        name: find-and-replace-version
+        run: |
+          VERSION=$(cat version.txt)
+          sed -i -r 's/[0-9]+.[0-9]+.[0-9]+(-dev)?/'"$VERSION"'/' cmd/cloud_sql_proxy/cloud_sql_proxy.go
```

Comment:
We probably want a slightly more strict regex for the entire line (`var versionString`) instead to make sure this doesn't change any where else we might want to have version numbers in the future.

Reply:
done